apps/partner2/test/index.coffee

```diff
@@ -10,7 +10,7 @@ Profile = require '../../../models/profile.coffee'
 newLayoutPartnerProfiles = _.map ['gallery_one', 'gallery_two', 'gallery_three'], (layout) ->
   new Profile fabricate 'partner_profile', owner: fabricate 'partner', profile_layout: layout
 
-deprecatedLayoutPartnerProfiles = _.map ['institution', 'gallery_default', 'gallery_deprecated'], (layout) ->
+deprecatedLayoutPartnerProfiles = _.map ['gallery_default', 'gallery_deprecated', 'institution'], (layout) ->
```

Comment:
I mean that we keep `institution` in deprecated profile, which is fine because the non admins should still see the old view. But we should add it to the new test for admins and verify that correct views get displayed in this case.